### DISCUSSION TITLE
Is it allowed to put up Blockchain-Ads on my public Github pages?

### DISCUSSION THREAD

**Discussion by tools-by-rotem** [community/community#146996] [Other Feature Feedback, Questions, & Ideas] [answered]
I saw the answers about Google Adsense putted up on github pages, yet I couldn't find any information about Blockchain-Ads.
Because of the sensitivity of the Blockchain and Crypto subjects, I want to be sure this is allowed.

**Comment by dineal22** [comment, marked as answer]
Placing blockchain-related ads on your public GitHub pages may violate GitHub's Terms of Service, which prohibit unsolicited commercial advertising, spam, and disruptive content. GitHub focuses on code sharing and collaboration, not on hosting ads. Ensure any content you post aligns with the platform's guidelines, and avoid promoting products or services unless they're directly related to your project. For more information visit http://templefade.com

**Comment by dineal22** [comment]
Advertising blockchain-related content on your public GitHub pages might breach GitHub's Terms of Service, which forbid unsolicited commercial ads, spam, and disruptive material. GitHub prioritizes code sharing and collaboration rather than serving advertisements. Make sure that any content you share adheres to the platform's rules, and refrain from promoting products or services unless they are directly connected to your project. For further details read [best blog](https://explorethings.org/)
